src/components/posts/PostHeader.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import logo from "../../assets/TeachLogo.png";
+
+function PostHeader(props) {
+  return (
+    <div style={styles.containerStyles}>
+      <img src={logo} alt="" style={styles.imgStyles} />
+      <div style={styles.headerTextSTyles}>{props.header}</div>
+
+      <div style={styles.subheaderTextStyles}>{props.subheader}</div>
+    </div>
+  );
+}
+const styles = {
+  imgStyles: {
+    float: "left",
+    width: "89px",
+    height: "83px",
+  },
+  containerStyles: {
+    backgroundColor: "white",
+    textAlign: "left",
+    width: "950px",
```

Comment:
Here and otherwise, I would avoid using pixel-fixed container sizes if possible, especially since you use them in many different areas. For the column, set `max-width: 1000px` (or `50rem`, etc.) somewhere once, and that should be it.

Reply:
Yup, we're going to be fixing the CSS/styling and converting it into class-based styling soon

Reply:
Shouldn't each post be the same width though, to make it look uniform?

Reply:
Nvm, it looks fine doing the max-width as you said